contentacms/contenta_jsonapi
# Issue 298: Remove media actions

Task: #298 


Relates to #298 

### Notes

Based on this issue: https://www.drupal.org/project/media_entity_actions/issues/2961959 for media_entity_actions

> Note however, that new sites are not required to install this module, unless they want to provide bulk actions for media entities (which are not in core). This module is a requirement for sites upgrading from Media Entity contrib because those actions were part of Media Entity and were removed in the core Media module. Existing sites that relied on those actions would miss them if this module is not available.

Based on that, I believe we can remove these actions to prevent config errors.

### Test Instructions

* Install contenta_jasonapi, completing Drupal install 
* /admin/config/development/configuration should not show the following after attempting to import changes
```
Unexpected error during import with operation create for system.action.media_delete_action: The "media_delete_action" plugin does not exist.
Unexpected error during import with operation create for system.action.media_publish_action: The "media_publish_action" plugin does not exist.
Unexpected error during import with operation create for system.action.media_save_action: The "media_save_action" plugin does not exist.
Unexpected error during import with operation create for system.action.media_unpublish_action: The "media_unpublish_action" plugin does not exist.
```

### QA
